tests/tasks.py

```diff
@@ -14,17 +16,17 @@ class FailValidateTask(Task):
     description = "this task always fails validation"
 
     @classmethod
-    def validate(self, item_collection):
+    def validate(self, payload: Dict[str, Any]) -> bool:
```

Comment:
@gadomski Might be out of scope for this PR but the README example doesn't match validate anymore. Maybe we can address that here or just create a separate ticket for that.

Reply:
Nah, let's get the docs in line, I'll fix. Thanks, good catch.